### PR TITLE
feat: update header nav

### PR DESCRIPTION
**Pull request by RichardIrala**
image

### CODE REVIEW

File: pages/inventory.js [hunk 1]
```diff
@@ -420,37 +420,48 @@ const Inventory = ({ activeChain, handleActiveChain }) => {
...
-              Connect with MetaMask
-            </button>
-          ) : (
-            <button className="inventory__button" onClick={logoutAndClearUI}>
-              logout
+      <nav className="global-styles__menu">
+        <ul>
+          <li>
+            <button className="global-styles__home__menuOption" onClick={goToHome}>
+              Go to home
```
Comment (luislucena16) [inline pages/inventory.js]: use `Go to Home`

File: pages/inventory.js [hunk 2]
```diff
@@ -420,37 +420,48 @@ const Inventory = ({ activeChain, handleActiveChain }) => {
...
-              console.error("Login Failed");
-            }}
-          /> */}
-        </div>
+          <li>
+            <button className="global-styles__home__menuOption">Active chain: {activeChain}</button>
+            <ul class="global-styles__home__subMenus">
+              <li>
+                <button className="global-styles__button__sub-menu__option" onClick={() => handleActiveChain(defaultTelosChainValue)}>
+                  Change to telos
```
Comment (luislucena16) [inline pages/inventory.js]: rename `telos` to `Telos`

File: pages/inventory.js [hunk 3]
```diff
@@ -420,37 +420,48 @@ const Inventory = ({ activeChain, handleActiveChain }) => {
...
+                </button>
+              </li>
+              <li>
+                <button className="global-styles__button__sub-menu__option" onClick={() => handleActiveChain(defaultPolygonChainValue)}>
+                  Change to polygon
```
Comment (luislucena16) [inline pages/inventory.js]: rename `polygon` to `Polygon`

File: pages/index.js [hunk 4]
```diff
@@ -158,16 +158,37 @@ export default function Home({ XerialWalletViewmodel, activeChain, handleActiveC
...
+              <button className="global-styles__home__menuOption" onClick={refreshListedItems}>
+                Refresh listed items
+              </button>
+            </li>
+            <li>
+              <button className="global-styles__home__menuOption">Active chain: {activeChain}</button>
+              <ul class="global-styles__home__subMenus">
+                <li>
+                  <button className="global-styles__button__sub-menu__option" onClick={() => handleActiveChain(defaultTelosChainValue)}>
+                    Change to telos
```
Comment (luislucena16) [inline pages/index.js]: rename `telos` to `Telos`

File: pages/index.js [hunk 5]
```diff
@@ -158,16 +158,37 @@ export default function Home({ XerialWalletViewmodel, activeChain, handleActiveC
...
+                  </button>
+                </li>
+                <li>
+                  <button className="global-styles__button__sub-menu__option" onClick={() => handleActiveChain(defaultPolygonChainValue)}>
+                    Change to polygon
```
Comment (luislucena16) [inline pages/index.js]: rename `polygon` to `Polygon`

File: pages/index.js [hunk 6]
```diff
@@ -158,16 +158,37 @@ export default function Home({ XerialWalletViewmodel, activeChain, handleActiveC
 
     return (
       <div className="home__principalContainer">
+        <nav className="global-styles__menu">
+          <ul>
+            <li>
+              <button className="global-styles__home__menuOption" onClick={goToInventory}>
+                Go to inventory
```
Comment (luislucena16) [inline pages/index.js]: use `Inventory`

File: pages/index.js [hunk 7]
```diff
@@ -158,16 +158,37 @@ export default function Home({ XerialWalletViewmodel, activeChain, handleActiveC
...
+              </button>
+            </li>
+            <li>
+              <button className="global-styles__home__menuOption" onClick={refreshListedItems}>
+                Refresh listed items
```
Comment (luislucena16) [inline pages/index.js]: use `Items`

**Review by luislucena16 — CHANGES_REQUESTED** [review]

File: pages/inventory.js (hunk 1 above)
Comment (RichardIrala) [reply pages/inventory.js]: Done!

File: pages/inventory.js (hunk 2 above)
Comment (RichardIrala) [reply pages/inventory.js]: Done!

File: pages/inventory.js (hunk 3 above)
Comment (RichardIrala) [reply pages/inventory.js]: Done!

File: pages/index.js (hunk 4 above)
Comment (RichardIrala) [reply pages/index.js]: Done!

File: pages/index.js (hunk 5 above)
Comment (RichardIrala) [reply pages/index.js]: Done!

File: pages/index.js (hunk 7 above)
Comment (RichardIrala) [reply pages/index.js]: Done

File: pages/index.js (hunk 6 above)
Comment (RichardIrala) [reply pages/index.js]: Done

**Review by luislucena16 — APPROVED** [review]

**Pull request merged**